ui/app/controllers/vault/cluster.js

```diff
@@ -24,19 +25,18 @@ export default Controller.extend({
 
   namespaceQueryParam: '',
 
-  /* eslint-disable-next-line ember/no-observers */
   onQPChange: observer('namespaceQueryParam', function() {
```

Comment:
This only needs to be an observer because it's based on a computed property, so we could potentially get away with computing on `clusterController.namespaceQueryParam` which is what namespaceQueryParam is an alias of